### PR TITLE
Add/edit comment on Ctrl+Enter

### LINKED ISSUE #352: Would be nice to use Ctrl+Enter to add comments
Like in GitHub, would be nice to be able to use Ctrl+Enter to add (rather, commit) comments.

### PR DESCRIPTION
**Pull request by heaths**
Fixes #352

### CODE REVIEW

**Comment by heaths** [comment]
Should we add a tooltip or something that lets people know this is possible, or just expect they'll try it if they are used to GitHub or other sites' comments?

**Comment by heaths** [comment]
There was a doc-warden failure for a missing file for python validation. @weshaggard who would be a good contact to look into this?

File: src/dotnet/APIView/APIViewWeb/Client/src/comments.ts [hunk 1]
```diff
@@ -90,6 +90,18 @@
         e.preventDefault();
     });
 
+    $(document).on("keydown", ".new-thread-comment-text", e => {
+        if (e.ctrlKey && e.keyCode === 13) {
+            const form = $(e.target).closest("form");
+            if (form) {
+                const button = form.children("[data-post-update='comments']");
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Client/src/comments.ts]: Ugh, I wonder if we can do something nicer. What about extracting the original `data-post-update` handler into a method and calling it from both places?

Comment (heaths) [reply src/dotnet/APIView/APIViewWeb/Client/src/comments.ts]: What about just replacing the form.submit handler?

Comment (pakrym) [reply src/dotnet/APIView/APIViewWeb/Client/src/comments.ts]: Try it, might look good.

**Comment by weshaggard** [comment]
>There was a doc-warden failure for a missing file for python validation. @weshaggard who would be a good contact to look into this?

Looks like that has been fixed but in the future it would be either @praveenkuttappan or @scbedd

**Pull request merged**